### ISSUE TITLE
[SUGGESTION] Admin Rights

### ISSUE THREAD

**Issue by RndUsr123** [austinnixholm/ThreeFingerDrag#14]
Elevated processes, like Task Manager, are unaffected by this tool when started as a normal user. This might not be immediately obvious so perhaps some instructions about this in the ReadMe might avoid confusing new users.

**Comment by austinnixholm** [comment]
Yeah this a great suggestion and has been on my mind for a little while. 

Version 1.3.0 is planned to feature an option to restart the program as administrator, as well as implement the [customization suggestions](https://github.com/austinnixholm/ThreeFingerDrag/issues/10) previously suggested. 

I'll additionally update the readme along with the 1.2.6 release which is right around the corner.
